pkg/codegen/internal/test/testdata/external-resource-schema/dotnet/Utilities.cs

```diff
@@ -60,6 +62,57 @@ public static InvokeOptions WithVersion(this InvokeOptions? options)
             };
         }
 
+        public static Input<Dictionary<string,T>> ToDict<T>(this InputMap<T> inputList)
+        {
+            return inputList.Apply(v => new Dictionary<string,T>(v));
+        }
+
+        public static List<T> ToList<T>(this IEnumerable<T> elements)
```

Comment:
Special casing for these in codegen is definitely a code smell; I wonder if there is any better way.